sbin/Release.sh

```diff
@@ -18,27 +18,32 @@ npm install
 # OpenJDK<version>_<arch>_<os>_<timestampOrTag>.<extension>
 # Examples: OpenJDK8_x64_Windows_201813060547.zip, 
 #           OpenJDK8_x64_LinuxLH_201813060547.tar.gz, 
-#           OpenJDK10_aarch64_Linux_201813060547.sha256.txt
+#           OpenJDK10_aarch64_Linux_201813060547.tar.gz.sha256.txt,
```

Comment:
seems a little odd that it has the tar.gz followed by the sha256.txt and then a `,`?  Is that all deliberate?  Looks like the sha256.txt suffix is...

Reply:
Yes this is correct.  I see that when sha256.txt files are created, they use the full filename (either OpenJDK10_aarch64_Linux_201813060547.tar.gz or OpenJDK8_x64_Windows_201813060547.zip and append the .sha256.txt suffix.  The ? is deliberate in the regex, because it means 0 or 1 instances of that suffix will be present in the files we may find in our for loop.

Reply:
From Jenkins configure view:
<img width="429" alt="screen shot 2018-06-15 at 10 42 33 am" src="https://user-images.githubusercontent.com/2836948/41473960-c4abde6e-7088-11e8-882b-e6cd26367b86.png">

Example resulting name of sha246.txt file:
<img width="502" alt="screen shot 2018-06-15 at 10 43 03 am" src="https://user-images.githubusercontent.com/2836948/41474010-db8e314a-7088-11e8-9d18-243435afbf79.png">

Reply:
This shell script is the same for all checksum jobs, the example above is showing the one from  openjdk8_openj9_build_checksum job.